app/main.go

```diff
@@ -117,6 +128,39 @@ func main() {
 		cleanupHandler(w, r, config)
 	})
 
+	// Run a sync loop for standalone use
+	if !env.IsAppEngine() {
+		log.Debug("Running outside of appengine, starting up a sync loop...")
+		ctx, cancel := context.WithCancel(context.Background())
+		count := 0
+		go func() {
```

Comment:
readability nit:

Can you move this to it's own func to make it easier to read?

Reply:
Done.